packages/fast-components-react-msft/src/heading/heading.spec.tsx

```diff
@@ -56,9 +48,30 @@ describe("heading", (): void => {
         expect(rendered.find(handledProps.tag).prop("aria-hidden")).toEqual(true);
     });
 
+    test("should render a default `tag` when no `tag` prop is passed", () => {
+        const rendered: any = mount(<MSFTHeading />);
+
+        expect(rendered.exists(HeadingTag.h1)).toBe(true);
+    });
+
     test("should render the correct `tag` when `tag` prop is passed", () => {
         const rendered: any = mount(<MSFTHeading tag={HeadingTag.h3} />);
 
         expect(rendered.exists(HeadingTag.h3)).toBe(true);
     });
+
+    test("should render the correct `size` when `size` prop is passed", () => {
```

Comment:
Can we have a test to render each valid size

Reply:
Working on adding this and others